tests/console/firewall_enabled.pm

```diff
@@ -16,30 +16,26 @@ use strict;
 use warnings;
 use testapi;
 use version_utils qw(is_upgrade is_jeos is_sle is_vmware is_leap is_tumbleweed);
+use serial_terminal 'select_serial_terminal';
 use Utils::Architectures;
 
 sub run {
     my ($self) = @_;
     if ($self->firewall eq 'firewalld') {
-        my $timeout = 30;
-        $timeout = 60 if is_ppc64le;
-        my $ret = script_run('firewall-cmd --state', timeout => $timeout);
-        if ($ret && is_upgrade && get_var('HDD_1') =~ /\b(1[123]|42)[\.-]/) {
-            # In case of upgrades from SFW2-based distros (Leap < 15.0 to TW) we end up without
-            # any firewall
-            record_soft_failure "boo#1144543 - Migration from SFW2 to firewalld: no firewall enabled";
-            $ret = 0;
+        assert_script_run("firewall-cmd --version", fail_message => "firewall-cmd is not present");
+        if (script_output('firewall-cmd --state', timeout => 60, proceed_on_failure => 1) !~ /running/) {
+            if (is_upgrade && get_var('HDD_1') =~ /\b(1[123]|42)[\.-]/) {
+                # In case of upgrades from SFW2-based distros (Leap < 15.0 to TW) we end up without any firewall
+                record_soft_failure "boo#1144543 - Migration from SFW2 to firewalld: no firewall enabled";
+                return;
+            }
+            if (is_jeos && is_vmware && (is_sle('15-SP4+') || is_leap('15.4+'))) {
```

Comment:
`is_jeos && (is_vmware || $the_cloud_image)` as SP4 is EOL

Reply:
or version check for 15.6+

Reply:
So cloud image comes without pre-installed firewall, hence it can be only jeos && vmware

Reply:
I think we can actually remove this `if` block after merging https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/21129 and instead not schedule the test module on vmware.